CHANGELOG.md

```diff
@@ -28,6 +28,10 @@
 
 #### Enhancements
 
+* Type name rules considers SwiftUI template code.
```

Comment:
this is now in the wrong section

Reply:
Should it start at Line 13 of this file?

Reply:
Yes